.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-latest, windows-latest, macos-13]
```

Comment:
macos-latest is now macos-14-arm64. setup-python does not support macos-arm64 yet.